api/server.js

```diff
@@ -0,0 +1,59 @@
+const express = require("express");
+
+const games = require("../games/gamesModel.js");
+
+const server = express();
+
+server.use(express.json());
+
+server.get("/games", async (req, res) => {
+  try {
+    const rows = await games.fetch();
+    res.status(200).json(rows);
```

Comment:
This looks good Angela, you're returning the `rows` (games) and setting the correct `HTTP status` 👍